docs/get-started/deployment-and-usage.md

```diff
@@ -99,6 +99,65 @@ this approach does not guarantee running once on every node. For example,
 tainted, non-ready nodes or some other reasons in Job scheduling may cause some
 node(s) will run extra job instance(s) to satisfy the request.
 
+### Deployment with Helm
+
+Node Feature Discovery Helm chart allow to easily deploy and manage NFD.
+
+#### Prerequisites
+
+[Helm package manager](https://helm.sh/) should be installed.
+
+#### Deployment with Helm
+
+To install the chart with the release name node-feature-discovery:
+
+```bash
+git clone https://github.com/kubernetes-sigs/node-feature-discovery/
+cd node-feature-discovery/deployment
+helm install node-feature-discovery ./node-feature-discovery/
+```
+
+The command deploys Node Feature Discovery on the Kubernetes cluster in the default configuration.
+The Configuration section describes how it can be configured during installation.
+
+#### Configuration
+
+You can override values from `values.yaml` and provide a file with custom values:
+
+```bash
+helm install node-feature-discovery ./node-feature-discovery/ -f values.yaml
```

Comment:
@ArangoGutierrez  you should create a `.yaml` file with overrides to fields for helm deployment then provide it with -f

is the comment here that the path is wrong and it should be `./values.yaml` ?

also the WARNING bit is related to your k8s deployment IMO (Openshift?)

Reply:
yeah don't mind the WARNING, I was pointing out that the path is wrong, and since this is user documentation, we should be more explicit so the user doesn't run into a `file not found` error log

Reply:
I see , thanks for clarifying !